src/main/java/loanbook/logic/SendReminder.java

```diff
@@ -0,0 +1,84 @@
+package loanbook.logic;
+
+import java.util.Date;
+import java.util.Properties;
+
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import loanbook.model.Model;
+import loanbook.model.loan.Loan;
+
+/**
+ * The class is used for create reminder email and send the email.
+ */
+public class SendReminder {
```

Comment:
This class looks very similiar to https://github.com/clk528/maven-spring/blob/master/src/main/java/com/clk/library/sendmail.java

If you referred to a particular code, do remember to provide credits.

Reply:
Ok ^-^